generators/fix-entity/index.js

```diff
@@ -116,12 +113,15 @@ module.exports = generator.extend({
          */
         const getLiquibaseFile = type => `${jhipsterVar.resourceDir}config/liquibase/changelog/${this.entityConfig.data.changelogDate}_added_${type}_${this.entityConfig.entityClass}.xml`;
 
+        // TODO: freeze object (safely)
```

Comment:
Actually we should `Object.freeze(myObject)` every time we can, but in this case it caused a weird bug. But note that if you have synchronous code, you should always [freeze](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) and even deep freeze your objects. It makes Javascript more reliable.